glossary.md

```diff
@@ -0,0 +1,172 @@
+# Glossary
+
+## Technical terms
+
+These are technical terms that appear in the course content and assessment criteria.
+
+This mostly exists as a reference, but is also worth looking through as part of your preparation for the End Point Assessment as the assessor may use these terms and will expect you to understand them.
+
+**Acceptance Criteria** - A set of requirements that must be met to mark a user story as complete. Often abbreviated to "AC".  
+_Module 6_
```

Comment:
I _assume_ that these italics mean "go look at this reading material for more info" and that learners would make the same guess, but is it worth us spelling that out somewhere?